employees/admin.py

```diff
@@ -7,7 +8,10 @@
 
 
 class TaskActivities(admin.ModelAdmin):
-    list_display = ("name",)
+    list_display = ("name", "is_active")
+
+    def has_delete_permission(self, request: HttpRequest, obj: TaskActivityType = None) -> bool:
```

Comment:
Function is not working with Optional[TaskActivityType]. None is necessary as without It i get error while accessing task activities via admin panel. Error  "has_delete_permission() missing 1 required positional argument: 'obj'". I also need to allocate a type to pass tests.

Reply:
So If `None` is needed here you should change typing to `Optional[TaskActivityType]` because other way it's not a valid typing once you can pass `None` value and you're writing differently.

Reply:
changed